lib/block_view.go

```diff
@@ -7166,25 +7518,54 @@ func (bav *UtxoView) _connectAcceptNFTBid(
 	// Sanity check that the royalties are reasonable and won't cause underflow.
 	if txMeta.BidAmountNanos < (creatorRoyaltyNanos + creatorCoinRoyaltyNanos) {
 		return 0, 0, nil, fmt.Errorf(
-			"_connectAcceptNFTBid: sum of royalties (%d, %d) is less than bid amount (%d)",
+			"_helpConnectNFTSold: sum of royalties (%d, %d) is less than bid amount (%d)",
 			creatorRoyaltyNanos, creatorCoinRoyaltyNanos, txMeta.BidAmountNanos)
 	}
 
 	bidAmountMinusRoyalties := txMeta.BidAmountNanos - creatorRoyaltyNanos - creatorCoinRoyaltyNanos
-
+	txn, txHash, verifySignatures := txMeta.Txn, txMeta.TxHash, txMeta.VerifySignatures
```

Comment:
Nit: I don't like relabels like this. They typically make the code harder to read. I would instead just rename these vars in all the places below.